Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'will_paginate-bootstrap'
 gem 'yt'
 gem 'yui-compressor'
 gem 'youtube_rails'
+gem 'google-api-client', '0.8'
```

Comment:
Can you try using latest version?
See https://github.com/google/google-api-ruby-client/blob/master/MIGRATING.md